Packs/OctoxLabs/README.md

```diff
@@ -0,0 +1,181 @@
+Octox Labs Cyber Security Asset Management platform
```

Comment:
Please delete the content of this file.

Reply:
[https://github.com/demisto/content/pull/21175/commits/02305ec65304b55b15e4b33696c4fe4b9e58703d](https://github.com/demisto/content/pull/21175/commits/02305ec65304b55b15e4b33696c4fe4b9e58703d)